std/algorithm/mutation.d

```diff
@@ -1896,6 +1930,9 @@ if (isRandomAccessRange!Range && hasLength!Range)
 Reverses $(D r) in-place, where $(D r) is a narrow string (having
 elements of type $(D char) or $(D wchar)). UTF sequences consisting of
 multiple code units are preserved properly.
```

Comment:
This doesn't necessarily have to be addressed in this PR, but this function does *not* properly handle reversal of sequences of combining characters. For example, reversing `ba\u0301d` ("bád") will result in `d\u0301ab` ("d́ab") instead of `da\u0301b` ("dáb"), which is likely *not* what the user would expect. This probably should be noted in a `Bugs:` section.